CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
 ```
 
-> **Note**: certain versions of npm (5-8) and Node.js (10-16) are required to run certain scripts.
-
 ---
```

Comment:
I can only click on the sidebar item text to expand/collapse despite the entire line being highlighted.

First part of the video shows `main` second part shows branch.

https://github.com/ionic-team/ionic-docs/assets/2721089/b845d67c-0373-426f-8407-b340ec2a867a

Reply:
I've removed the hover styling to reflect main. Since it's no longer showing highlighted, would you still expect to click anywhere on the line or would you expect to click where the cursor is a pointer?

Reply:
I would expect the behavior to match what's on the production website. Note that you can click anywhere on the child links.